charts/lakefs/templates/fluffy-deployment.yaml

```
@@ -0,0 +1,123 @@
+# Configuration in this file is related to lakeFS enterprise solutions, which are outside of the open source scope. When running regular open-source installation the resources in this file should be used.
+{{- if (.Values.fluffy).enabled }}
+
+apiVersion: apps/v1
```

remove empty whitespace

```suggestion
{{- if (.Values.fluffy).enabled }}
apiVersion: apps/v1
```